core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -234,10 +234,25 @@ public boolean isLaunchSupported() {
         return launcher.isLaunchSupported();
     }
 
+    /**
+     * Return the {@code ComputerLauncher} for this SlaveComputer.
+     */
     public ComputerLauncher getLauncher() {
         return launcher;
     }
 
+    /**
+     * Return the {@code ComputerLauncher} for this SlaveComputer, strips off
+     * any {@code DelegatingComputerLauncher}s.
+     */
```

Comment:
`@since TODO`

Reply:
Seems like the next version is 2.80, correct?

Reply:
Well, the train departs today. Hard to say if it gets into this release or not